api/src/main/resources/messages.properties

```diff
@@ -56,6 +56,12 @@ coreapps.task.mergeVisits.mergeSelectedVisits=Merge Selected Visits
 coreapps.task.mergeVisits.success=Visits merged successfully
 coreapps.task.mergeVisits.error=Failed to merge visits
 
+coreapps.task.deletePatient.label=Delete Patient
+coreapps.task.deletePatient.notAllowed=User does not have permission to delete patient
+coreapps.task.deletePatient.deletePatientSuccessful=Patient has been successfully deleted
```

Comment:
You can actually go ahead to fix this message too as you fix the void reason issue too